src/app/containers/InlineContainer/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from 'react';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import Blocks from '../Blocks';
+import fragment from '../Fragment';
+import InlineLink from '../InlineLink';
+import InlineSpan from '../../components/InlineSpan';
+import inlineBlockPropTypes from '../../models/propTypes/inline';
+import { GridItemConstrainedMedium } from '../../lib/styledGrid';
+
+const componentsToRender = { fragment, urlLink: InlineLink };
+
+const InlineContainer = ({ blocks, language }) => {
+  const { script, service } = useContext(ServiceContext);
+
+  return (
+    <GridItemConstrainedMedium>
+      <InlineSpan lang={language} script={script} service={service}>
+        <Blocks blocks={blocks} componentsToRender={componentsToRender} />
```

Comment:
So when this renders in the client, we get the following structure:
```
div.ItemConstrainedMedium
  p.Paragraph
    div.ItemConstrainedMedium
      span.InlineSpan
```
It's invalid HTML to nest a `div` in a ` p`, and as such, this error is thrown in the console
```
react-dom.development.js?61bb:506 Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
```